src/test/python/parser/opentrackio_lib.py

```diff
@@ -79,53 +187,42 @@ def validate_dict_elements(self, mdict, keys):
             else:
                 return False
         return True
-
-    def conversion_factor_from_meters(self,unit_str):
-        """Stores the multiplicative conversion factor from meters to user-specified units
-        Valid unit specifier strings: m, cm, mm, in"""
-        if unit_str == "m":
-            return 1.0
-        elif unit_str == "cm":
-            return 100.0
-        elif unit_str == "mm":
-            return 1000.0
-        elif unit_str == "in":
-            return (1000.0/25.4)
-
-    def get_camera_translation(self, dimension):
-        """Return a single axis of camera translation, such as: x,y, or z"""
```

Comment:
You'l need to add this string back in, as it was omitted below.  (Or the docs won't have anything for this method.)